js/app.js

```diff
@@ -32,6 +36,23 @@ function toTaggedBase64(tag, data) {
         .replace(/\//g, "_");
 }
 
+function fromTaggedBase64(tb64) {
+    const [tag, dataCs] = tb64.split("~");
+    if (typeof dataCs == 'undefined') {
+        return [];
```

Comment:
My intuition is that `null` or `undefined` would be a better error sentinel than `[]` (since `[...]` in this case is really not a list but a pair). But I know JS is JS so take this as a just a light suggestion

Reply:
Good suggestion. I was assuming that this code is merely to show how it's done. I'd like to revisit this along with better testing.